README.md

```diff
@@ -705,6 +705,40 @@ Both these settings (`strictOptionParsing` and `strictArrayExtraction`) can be e
 val formats: Formats = DefaultFormats.strict
 ```
 
+Json 3.6 expands the `strictOptionParsing` behavior to throw a MappingException even when `extractOpt` is called on a missing optional attribute.  This behavior might not be desirable if you have a large code base that was using `extractOpt` prior to Json 3.6.  If you prefer to maintain the pre-3.6 `extractOpt` behavior, you can specify the format option `strictOptionParsingPre36`
```

Comment:
(A by-stander here)
Shouldn't the flag be called with respect to what it does. In two years "no-one" will care for what was the behavior before `3.6`

Reply:
@BlueTufa?

Reply:
How about `strictOptionParsingIgnoreMissing`?